spec/std/process_spec.cr

```diff
@@ -17,8 +17,8 @@ describe Process do
 
   it "raises if command could not be executed" do
     # FIXME: Oddly doubled error message
-    expect_raises_errno(Errno::ENOENT, "execvp: No such file or directory: No such file or directory") do
-      Process.new("foobarbaz")
+    expect_raises_errno(Errno::ENOENT, %(execvp (foobarbaz "foo"): No such file or directory: No such file or directory)) do
```

Comment:
Does the printed command need a length restriction? If it has many arguments, then there could be some issues with the usability.

Reply:
Restricting the length could also have usability implications. The best solution would be to change the order and print the failed command after the libc error message. `Errno` doesn't support that though.